Kitodo/src/main/java/org/kitodo/production/forms/MigrationForm.java

```diff
@@ -364,28 +365,30 @@ public void useExistingTemplate(Template template, Template existingTemplate) {
      *            The template to create.
      */
     public void createNewTemplate(Template template) {
-        try {
-            Converter converter = new Converter(template.getWorkflow().getTitle());
-            converter.convertWorkflowToTemplate(template);
-        } catch (IOException | DAOException | WorkflowException e) {
-            Helper.setErrorMessage(ERROR_SAVING, new Object[] {ObjectType.PROCESS.getTranslationSingular() }, logger,
-                    e);
-        }
+        if (isTitleValid(template)) {
```

Comment:
Could the content of this method be moved to `MigrationService` to keep `MigrationForm` free of program logic?